### PR TITLE
Fix15513

### PR DESCRIPTION
**Pull request by puneet**
Fix for https://issues.dlang.org/show_bug.cgi?id=15513

### CODE REVIEW

File: src/rt/sections_elf_shared.d [hunk 1]
```diff
@@ -200,6 +200,10 @@ version (Shared)
         assert(_loadedDSOs.empty);
         _loadedDSOs.swap(*cast(Array!(ThreadDSO)*)p);
         .free(p);
+        foreach(ref dso; _loadedDSOs) {
```
Comment (andralex) [inline src/rt/sections_elf_shared.d]: space before`(` :)

**Comment by andralex** [comment]
lgtm... @MartinNowak ?

**Comment by puneet** [comment]
Rebased fix15513 from the master.

I am new to team collaboration on github. Do not know if I did the right thing.

**Comment by MartinNowak** [comment]
Looks about right.
Adding the test case would be great https://github.com/D-Programming-Language/druntime/tree/master/test/shared.

**Comment by schuetzm** [comment]
Something went wrong with your rebase... On success, you should only see commits by yourself in the list, preferably only one for such a small change.

**Comment by puneet** [comment]
Yes, something seems to be wrong. I am closing this pull request and opening a new one.

**Comment by puneet** [comment]
Sorry for the mess. Here is the revised PR
https://github.com/D-Programming-Language/druntime/pull/1507

**Pull request closed without merging**